### DISCUSSION TITLE
'Submit Request' button is missing/chopped off after clicking Request a Change to this listing

### DISCUSSION THREAD

**Discussion by namdrino** [code4recovery/12-step-meeting-list#533] [Q&A] [answered]
For some reason the grey box to submit your change request has been cut off and is barely visible.
![changebox](https://user-images.githubusercontent.com/72889483/134724971-9fad5727-d1d6-41ef-9f0e-1aa8311a952c.jpg)

Any ideas how to fix?

**Comment by kiyote33** [comment]
More info on the platform/browser would help for us to replicate the issue. From what I see, it looks like the font is over-sized. On most Window's browsers you can reduce the font by simply  holding the Ctrl button down and scrolling the mouse button down.  If that doesn't work, can you let us know a bit more about the particulars of your situation?

**Reply by tech2serve** [reply]
To echo @kiyote33, we’re going to need more information to help answer this.

In my Safari, when I increased the font size, my theme (responsive) made the entire block bigger, and the submit button was still present.

If playing with the font size doesn’t fix things, please let us know what theme you’re using, and open up a debugging console to see if any errors are thrown. Sometimes weird things like this occur with Wordpress because of conflicts with other plugins.

Please let us know. We’d like to help you get this sorted out.

**Comment by namdrino** [comment]
Thanks for the help.  So the problem happens in both Safari on Mac as well
as iOS.  I don't think it's a font size issue.  You can see the issue live
by going to https://birminghamaa.org/wp/meetings/early-risers/ and clicking
Request a Change to this Listing.    Theme is called Solar Energy.   Next
step I'll open up a debugging console and see if any errors show.

**Comment by namdrino** [comment, marked as answer]
I was able to fix it via CSS.  Thanks anyways !

**Reply by tech2serve** [reply]
Super. It was probably theme related then. 

I see what theme you’re using. I may try and dig into this deeper later this weeken.

Glad you got it the way you wanted!
